docs/modules/user-guide/partials/proc_adding-a-name-to-a-devfile.adoc

```diff
@@ -22,15 +22,15 @@ metadata:
 ----
 ====
 
-. To specify a prefix for automatically generated workspace names, define the `generateName` attribute and don't define the `name` attribute. The workspace name will be in the `<generateName>YYYYY` format (for example, `che-2y7kp`). `Y` is random `[a-z0-9]` character.
+. To specify a prefix for automatically generated workspace names, define the `generateName` attribute and don't define the `name` attribute. The workspace name will be in the `<generateName>YYYYY` format (for example, `devfile-sample-2y7kp`). `Y` is random `[a-z0-9]` character.
 +
 .Adding a generated name to a devfile
 ====
```

Comment:
No need for this extra block. Remove these blocks everywhere (I only suggested the change in a handful of examples, but they are present in many places).
```suggestion
```